### ISSUE TITLE
Small landing zone for tear off

### ISSUE THREAD

**Issue by NottocD** [ButchersBoy/Dragablz#139]
Using your sample application (Dragablz-master V0.3.182) compiled with 2015 .45 start up project. I first combined all the dialogs together like this:
![dragerror1](https://cloud.githubusercontent.com/assets/18472529/21556211/5cdc11a2-cde4-11e6-973b-6c7fbfbd1117.jpg)
I next split off two of the tabs together onto a separate window like this:
![dragerror2](https://cloud.githubusercontent.com/assets/18472529/21556337/405ba08c-cde5-11e6-8ed6-c6c21b42c1c1.jpg)
I found when trying to add the second window the landing zone is very small if I want them to be together as shown. Is there a way to increase the size of the landing zone? I notice the other two sample windows have a much larger landing zone, is this because the first tab isn't fixed for those others?

**Comment by ButchersBoy** [comment]
You mean the semi circle landing zones at the side, right?

**Comment by NottocD** [comment]
No, I'm not referring to the docking landing zone semi-circle. I attached a small movie. Notice that right before the second tab (July) joins with the first torn off tab (Header) the semi-circles disappear, and the July tab positions as it's part of the tab collection. At which point I let go of the mouse button and July and Header are joined together. I'm talking about the landing zone that appears to be located on the rightish side of the Header tab. Is that landing zone adjustable?

[bandicam 2017-01-12 15-05-03-558.zip](https://github.com/ButchersBoy/Dragablz/files/702899/bandicam.2017-01-12.15-05-03-558.zip)
